### PR TITLE
Do not warn about 0.5.0 experimental pragma.

### PR DESCRIPTION
**Pull request by chriseth**
Closes https://github.com/ethereum/solidity/issues/3576

Depends on #3599.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Features:
  * Type Checker: Disallow uninitialized storage pointers as experimental 0.5.0 feature.
  * Support and recommend using ``emit EventName();`` to call events explicitly.
  * Syntax Analyser: Do not warn about experimental features if they do not concern code generation.
+ * Syntax Analyser: Do not warn about `pragma experimental "v0.5.0"`.
```
Comment (axic) [inline Changelog.md]: I would extend it: "since they do not affect code generation". I know it is stated above, but better to be explicit.

**Comment by chriseth** [comment]
Changed.

File: Changelog.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ Features:
  * Type Checker: Disallow uninitialized storage pointers as experimental 0.5.0 feature.
  * Support and recommend using ``emit EventName();`` to call events explicitly.
  * Syntax Analyser: Do not warn about experimental features if they do not concern code generation.
+ * Syntax Analyser: Do not warn about `pragma experimental "v0.5.0"` since it does not affect code generation.
```
Comment (axic) [inline Changelog.md]: Needs double backticks.

**Comment by chriseth** [comment]
Updated.

**Review by axic — APPROVED** [review]
Looks good (obviously we need to merge #3599 first).

**Comment by chriseth** [comment]
No, it's the other way round.

**Comment by axic** [comment]
> No, it's the other way round.

I don't think so, #3599 removed the last non-analysis-only 0.5.0 change.

**Comment by chriseth** [comment]
ah that way! sorry!

**Pull request merged**